common/src/test/java/org/apache/falcon/util/ApplicationPropertiesTest.java

```diff
@@ -39,6 +40,16 @@
     private static final String PROPERTY_2 = "property-key-2";
     private static final String JKS_FILE_NAME = "credentials.jks";
 
+    @AfterClass
+    public void tearDown() throws Exception {
+        // delete temporary jks files
+        File credDir = new File(".");
```

Comment:
Can we define credDir as public static final String CRED_DIR so that it can be reused in tearDown and testResolveAlias? Thanks!